proto/ligato/vpp/wireguard/wireguard.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package ligato.vpp.wireguard;
+
+option go_package = "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/wireguard;vpp_wg";
+
+message Peer {
+  // Public-key base64
+  string public_key = 1;
+
+  // Peer UDP port
+  uint32 port = 2;
+
+  // Keepalive interval (sec)
+  uint32 persistent_keepalive = 3;
+
+  //  Table ID
+  uint32 table_id = 4;
```

Comment:
`Table ID` is most likely referencing to VRF table, so this should be named `vrf_id` to be consistent with other proto models.